docs/src/numerical_implementation/boundary_conditions.md

```diff
@@ -123,3 +123,103 @@ boundaries ``\partial \Omega_b``:
    values of ``c``.
 
 Flux boundary conditions are represented by the [`Flux`](@ref) type.
+
+## Open boundary conditions
+
+Open boundary conditions directly specify the value of the halo points. Typically this is used
+to impose no penitration boundary conditions, i.e. setting wall normal velocity components on 
```

Comment:
This is spelled with an `e`, no?

```suggestion
to impose no penetration boundary conditions, i.e. setting wall normal velocity components on 
```

This occurs a few other times throughout this file.